tests/libtest/correct.stdout

```diff
@@ -1,84 +1,84 @@
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/rule.feature::12: Rule: rule::14: Scenario: 2 secs::17:  Then 2 secs"}
+{"type":"test","event":"failed","name":"Feature: Basic tests/features/wait/rule.feature::12: Rule: rule::14: Scenario: 2 secs::17:  Then 2 secs","stdout":"Defined: tests/features/wait/rule.feature:17:7\nMatched: tests/libtest.rs:9:1\nStep panicked. Captured output: Too much!\nWorld(\n    4,\n)\n"}
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/nested/rule.feature::6: Scenario: 1 sec::3: Background Given 1 sec"}
+{"type":"test","event":"ok","name":"Feature: Basic tests/features/wait/nested/rule.feature::6: Scenario: 1 sec::3: Background Given 1 sec"}
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/nested/rule.feature::6: Scenario: 1 sec::7:  Given 1 sec"}
+{"type":"test","event":"ok","name":"Feature: Basic tests/features/wait/nested/rule.feature::6: Scenario: 1 sec::7:  Given 1 sec"}
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/nested/rule.feature::6: Scenario: 1 sec::8:  When 1 sec"}
+{"type":"test","event":"ok","name":"Feature: Basic tests/features/wait/nested/rule.feature::6: Scenario: 1 sec::8:  When 1 sec"}
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/nested/rule.feature::6: Scenario: 1 sec::9:  Then unknown"}
+{"type":"test","event":"ignored","name":"Feature: Basic tests/features/wait/nested/rule.feature::6: Scenario: 1 sec::9:  Then unknown"}
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/nested/rule.feature::12: Rule: rule::14: Scenario: 2 secs::3: Background Given 1 sec"}
+{"type":"test","event":"ok","name":"Feature: Basic tests/features/wait/nested/rule.feature::12: Rule: rule::14: Scenario: 2 secs::3: Background Given 1 sec"}
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/nested/rule.feature::12: Rule: rule::14: Scenario: 2 secs::15:  Given 2 secs"}
+{"type":"test","event":"ok","name":"Feature: Basic tests/features/wait/nested/rule.feature::12: Rule: rule::14: Scenario: 2 secs::15:  Given 2 secs"}
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/nested/rule.feature::12: Rule: rule::14: Scenario: 2 secs::16:  When 2 secs"}
+{"type":"test","event":"ok","name":"Feature: Basic tests/features/wait/nested/rule.feature::12: Rule: rule::14: Scenario: 2 secs::16:  When 2 secs"}
+{"type":"test","event":"started","name":"Feature: Basic tests/features/wait/nested/rule.feature::12: Rule: rule::14: Scenario: 2 secs::17:  Then 2 secs"}
+{"type":"test","event":"failed","name":"Feature: Basic tests/features/wait/nested/rule.feature::12: Rule: rule::14: Scenario: 2 secs::17:  Then 2 secs","stdout":"Defined: tests/features/wait/nested/rule.feature:17:7\nMatched: tests/libtest.rs:9:1\nStep panicked. Captured output: Too much!\nWorld(\n    4,\n)\n"}
```

Comment:
@ilslv unfortunately, this way we lost intra-IDE links to the exact code sourc which are super-comfy!
<img width="502" alt="Screenshot 2022-07-24 at 16 56 04" src="https://user-images.githubusercontent.com/7114909/180650456-d10606ee-da06-45c4-8ae6-d6c2f39cbdef.png">

But, we may change this to preserve them.
<img width="502" alt="Screenshot 2022-07-24 at 17 02 12" src="https://user-images.githubusercontent.com/7114909/180650631-51004c54-51eb-425a-b605-052d8bd566a4.png">